tools/read_data_files/README.md

```diff
@@ -0,0 +1,28 @@
+# read_data_files
+
+`read_data_files` is a utility to extract data for all timeseries' present in a TSDB file set.
+
+# Usage
+```
+$ git clone git@github.com:m3db/m3db.git
+$ make tools
+$ ./bin/read_data_files
+Usage: read_data_files [-b value] [-n value] [-p value] [-s value] [parameters ...]
+ -b, --block-start=value
+       Block Start Time [in nsec]
+ -f, --id-filter=value
+       ID Contains Filter [e.g. xyz]
+ -n, --namespace=value
+       Namespace [e.g. metrics]
+ -p, --path-prefix=value
+       Path prefix [e.g. /var/lib/m3db]
+ -s, --shard-id=value
```

Comment:
I think our other tools does just call this shard? If so, would be nice to be consistent

Reply:
sure thing

Reply:
turns out we had a weird mix of shard and shard-id in various tools, standardised to `shard` as suggested.